airbyte-commons-server/src/main/java/io/airbyte/commons/server/services/AirbyteRemoteOssCatalog.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.commons.server.services;
+
+import io.airbyte.commons.constants.AirbyteCatalogConstants;
+import io.airbyte.config.CatalogDefinitionsConfig;
+import io.airbyte.config.init.RemoteDefinitionsProvider;
+import io.airbyte.config.specs.CombinedConnectorCatalogDownloader;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Convenience class for retrieving the remote OSS Catalog.
+ */
+// This should be deleted, in favour of the DefinitionsProvider Singleton when cloud is migrated to
+// micronaut
+@Deprecated(forRemoval = true)
+@SuppressWarnings("PMD.AvoidCatchingThrowable")
```

Comment:
Just curious: What is this suppressing?

Reply:
I am not sure..

This was from the old class.

ill remove, see what happens.